Tests/YCoreUITests/Protocols/SystemImageTests.swift

```diff
@@ -33,6 +33,12 @@ final class SystemImageTests: XCTestCase {
         YCoreUI.isLoggingEnabled = true
     }
     
+    func test_loadImageWithRenderingMode() {
+        RenderMode.allCases.forEach {
+            XCTAssertNotNil($0.loadImage())
```

Comment:
This test doesn't test anything related to render mode? Think about what should be happening in this case and what we should be asserting for.